stac_ipyleaflet/core.py

```diff
@@ -61,34 +60,46 @@ def __init__(self, **kwargs):
         self.marker_added = False
 
         self.create_buttons_layout()
-        self.add_biomass_layers_options()
+        self.check_for_env_vars()
+        self.add_default_layer_options()
         self.add_custom_tools()
 
         self.point_control = InspectControlWidget.template(self)
         self.draw_control = DrawControlWidget.template(self)
         return None
 
+    def check_for_env_vars(self):
+        required = [TITILER_STAC_ENDPOINT, TITILER_ENDPOINT]
+        if None in required:
+            logging.error("Missing required environment variable(s)")
+        return
```

Comment:
@emmalu 
This is a breaking change now as we not longer default! Do we want to push up the `.env` file??? I dont think we should... but when we deploy this to each workspace env, there needs to be a `.env` file bundled in... to be honest i'm not sure how to do that... I can look into it though. But for example... Locally, we should now have a `.env` file at the root. And when this is deployed to whichever environment (MAAP or VEDA) these envs would have their own envs, which i'm wondering do we specify in a build?

Reply:
Could we still default to use values that point to MAAP resources? I'm only suggesting do default as a fall-back; in the event that an `.env` file goes missing? Or to be able to test in Google Co-Lab or Binder?

And yes, I think we would specify the `.env` file to use at build time - so that everything is packaged together in the forthcoming image update.

Reply:
📓 : Spoke to emma about this, this is not breaking change, but will log a warning if any env vars are missing. Next steps are to figure out the deployment process to bundle .env file